File: 217-contains-duplicate/contains-duplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {


        Set<Integer> hs = new HashSet<>();

        for(int num : nums)
        {
            if(!hs.contains(num))
            {
                hs.add(num);
            }else{
                return true;
            }
        }

        return false;
        
    }
}

File: 217-contains-duplicate/contains-duplicate.kt
class Solution {
    fun containsDuplicate(nums: IntArray): Boolean {

        val set = mutableSetOf<Int>()
        

        for(i in nums)
        {
            if(set.contains(i))
            {
                return true
            }else{
                set.add(i)
            }
        }

        return false;
    }
}